src/main/native/cpp/Main.cpp

```diff
@@ -2,6 +2,8 @@
 // Open Source Software; you can modify and/or share it under the terms of
 // the WPILib BSD license file in the root directory of this project.
 
+#include <imgui.h>
```

Comment:
Why was this added? Should it be moved down with the other includes?

Reply:
I'm not too sure as to why it was added. I think VSCode might've done it on its own?

Reply:
Since we're using imgui calls directly, we should probably keep it but sort it with the other headers.